sdk/identity/identity/src/credentials/azureCliCredential.ts

```diff
@@ -22,8 +22,12 @@ function getSafeWorkingDir(): string {
 const logger = credentialLogger("AzureCliCredential");
 
 /**
- * Provides the user access token and expire time
+ * This credential will use the currently logged-in user login information
```

Comment:
Re-wording to remove the repeated reference to the "log in" concept. Since we anyway mention that you need to be logged in towards the end, I believe this edit should be ok

```suggestion
 * This credential will use the user login information
```